src/core/stdcpp/new_.d

```diff
@@ -0,0 +1,98 @@
+/**
+ * D binding to C++ <new>
+ *
+ * Copyright: Copyright (c) 2019 D Language Foundation
+ * License: Distributed under the
+ *      $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0).
+ *    (See accompanying file LICENSE)
+ * Authors:   Manu Evans
+ * Source:    $(DRUNTIMESRC core/stdcpp/new_.d)
+ */
+
+module core.stdcpp.new_;
+
+import core.stdcpp.xutility : __cpp_sized_deallocation, __cpp_aligned_new;
+
+// TODO: this really should come from __traits(getTargetInfo, "defaultNewAlignment")
+enum size_t __STDCPP_DEFAULT_NEW_ALIGNMENT__ = 16;
```

Comment:
Is this necessary to be public?

Reply:
Is it conceivable that someone binds their own containers that respond to this value?
It does need to be a `getTargetInfo` though; this value is wrong.

Reply:
I don't know enough about this but it looks like implementation details. BTW, they miss Ddoc comments.

Reply:
Yeah, I didn't put ddoc because I wasn't sure if public or not.

Reply:
Either have them as public and documented or not public.